test/NuGet.Core.Tests/NuGet.CommandLine.Xplat.Tests/Utility/XPlatUtilityTests.cs

```diff
@@ -37,5 +43,36 @@ public void MSBuildVerbosityToNuGetLogLevel_HasProperMapping(string verbosity, L
 
             Assert.Equal(expected, actual);
         }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData(null)]
+        public void ProcessConfigFile_GetSettingsForWorkingDirectory(string emptyConfig)
+        {
+            ISettings settings = XPlatUtility.ProcessConfigFile(emptyConfig);
+            var baseDirectory = NuGetEnvironment.GetFolderPath(NuGetFolderPath.UserSettingsDirectory);
+            string baseNugetConfigPath = Path.Combine(baseDirectory, Settings.DefaultSettingsFileName);
+            List<string> configPaths = settings.GetConfigFilePaths().ToList();
+            Assert.True(configPaths.Count > 1);
+            // Assert user default nuget.config is loaded
+            Assert.True(configPaths.Contains(baseNugetConfigPath));
```

Comment:
Since command doesn't set specific working directory itself, it's just test binary folder, so several nuget.config including user default nuget.config'll get loaded.

Reply:
Put that comment in the code.

Reply:
Sure, I added it.